Countries Info/Helpers/HelpersButtons.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  HelpersButtons.swift
+//  Countries Info
+//
+//  Created by Oleh Stasiv on 11.12.2020.
+//
+
+import UIKit
+
+class CurrenciesButton: UIButton {
```

Comment:
nit: You can use there something like this **class TimeZonesButton: UIButton { }** to save some space.